cmd/corimDisplay_test.go

```diff
@@ -60,23 +68,28 @@ func Test_CorimDisplayCmd_bad_signed_corim(t *testing.T) {
 	require.NoError(t, err)
 
 	err = cmd.Execute()
-	assert.EqualError(t, err, "error decoding signed CoRIM from bad.txt: failed CBOR decoding for COSE-Sign1 signed CoRIM: cbor: invalid COSE_Sign1_Tagged object")
+	assert.EqualError(t, err, "error decoding CoRIM (signed or unsigned) from bad.txt: expected map (CBOR Major Type 5), found Major Type 3")
 }
 
-func Test_CorimDisplayCmd_invalid_signed_corim(t *testing.T) {
+func Test_CorimDisplayCmd_invalid_signed_corim_Truncated(t *testing.T) {
 	cmd := NewCorimDisplayCmd()
 
 	args := []string{
-		"--file=invalid.cbor",
+		"--file=truncated.cbor",
 	}
 	cmd.SetArgs(args)
 
 	fs = afero.NewMemMapFs()
-	err := afero.WriteFile(fs, "invalid.cbor", testSignedCorimInvalid, 0644)
```

Comment:
What about the invalid.cbor testdata is unsuitable now? This orphans the test file and embedding variable.

Reply:
@deeglaze yee, i kinda missed this will update this..

Reply:
@deeglaze I have updated the test. I really appreciate your help and guidance, especially while you are on holiday. Thank you so much for taking the time to assist me—it means a lot!